roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -115,3 +115,15 @@
   when:
     - openshift_master_console_port is defined
     - openshift_master_console_port != openshift_master_api_port
+
+- name: At least one master is schedulable
```

Comment:
task name is misleading.
name: All masters are schedulable
from the logic "false in l_master_schedulable"